docs/_includes/extra/script.js

```diff
@@ -2,6 +2,26 @@
 *  doesn't work due to fixed body height when menu is toggled.
 *  This script intercepts clicks on links, toggles the menu off
 *  and performs the anchor navigation. */
+
+window.addEventListener("load", function () {
+    let href = window.location.pathname;
+    const hash = window.location.hash;
+    if (hash!=="") {
+        href=hash
+    }
```

Comment:
nit: would be nice to have consistent formatiing

Reply:
Hm, as I can see, all code has a tab as a separator. formatting is good

Reply:
spaces before comparison operators are omitted here. While in the rest of the code there is always space before and after comparison operators

Reply:
fixed